Sources/Bitski-iOS-SDK/Core/Bitski.swift

```diff
@@ -353,13 +361,14 @@ public class Bitski: NSObject, BitskiAuthDelegate {
             completion(nil, AuthenticationError.notLoggedIn)
             return
         }
-        authState.performAction { [weak self](accessToken, _, error) in
+        authState.performAction { [weak self](accessToken, idToken, error) in
             guard let self = self else {
                 completion(nil, error)
                 return
             }
-            if let accessToken = accessToken {
+            if let accessToken = accessToken, let idToken = idToken {
                 self._authToken = accessToken
+                self._idToken = idToken
                 completion(accessToken, nil)
```

Comment:
How do we get the _idToken out? Just returning access token here.

Reply:
@ptescher     public var idToken: String {
       return _idToken
    } 

?

Reply:
so you call getCurrentAccessToken() first, then call .idToken ?

Reply:
yes _idToken will piggyback that call then the public variable will return the private _idToken I don't think it is needed for signing transactions. wdyt?

Reply:
fine for now

Reply:
@ptescher is it ok for now ?